comments/functions.js

```diff
@@ -0,0 +1,52 @@
+$(document).ready(function(){
+
+   $(".comments").sortable({
+      revert: true
+    });
+
+   $(document).on("click", "input[name=hide-form]", function(){
```

Comment:
why do you use search in all document? i think you can use small context, like: $comments.on("click", "input[name=hide-form]", ...